Source/Core/DolphinQt/GCMemcardCreateNew.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/GCMemcardCreateNew.h"
+
+#include <QComboBox>
+#include <QFileDialog>
+#include <QGridLayout>
+#include <QLabel>
+#include <QPushButton>
+#include <QRadioButton>
+
+#include "Common/FileUtil.h"
+#include "Common/MsgHandler.h"
+
+#include "Core/HW/GCMemcard/GCMemcard.h"
+
+GCMemcardCreateNew::GCMemcardCreateNew(QWidget* parent) : QDialog(parent)
+{
+  m_combobox_size = new QComboBox();
+  m_combobox_size->addItem(tr("4 Mbit (59 blocks)"), QVariant(4));
+  m_combobox_size->addItem(tr("8 Mbit (123 blocks)"), QVariant(8));
+  m_combobox_size->addItem(tr("16 Mbit (251 blocks)"), QVariant(16));
+  m_combobox_size->addItem(tr("32 Mbit (507 blocks)"), QVariant(32));
+  m_combobox_size->addItem(tr("64 Mbit (1019 blocks)"), QVariant(64));
+  m_combobox_size->addItem(tr("128 Mbit (2043 blocks)"), QVariant(128));
+  m_combobox_size->setCurrentIndex(5);
+
+  m_radio_western = new QRadioButton(QString(tr("Western (Windows-1252)")));
```

Comment:
`tr` already returns a QString
```suggestion
  m_radio_western = new QRadioButton(tr("Western (Windows-1252)"));
```